examples/1.computer_vision/image_classification/image_classification.ipynb

```diff
@@ -1,11 +1,11 @@
 {
```

Comment:
<p>Let's use pip instead of pip3 for consistency</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>